.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+permissions:
+  contents: read
+
+jobs:
+  goreleaser:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
For security purposes, we should set the SHA instead the version: [docs](https://enghub.grafana-ops.net/docs/default/component/deployment-tools/platform/continuous-integration/#pin-actions-to-a-full-length-commit-sha). Its something that we need to update in other repos 😅. Here you are an [example](https://github.com/grafana/deployment_tools/blob/master/.github/actions/kube-manifests-exporter/conftest-full/action.yml#L34).

Reply:
TIL! Thanks for sharing that @spinillos. I agree, let's pin to a specific version here and we can clean up other instances in another PR

Reply:
I didn't know that, thank you! Changed it for the new actions, did the same for other actions that we use, and updated the docs :)